.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md
```

Comment:
- [ ] Great job implementing the required change here. But your `test.md` file is still showing up on Github. In order to fix this, you could try the [solutions on this thread](https://stackoverflow.com/questions/45400361/why-is-gitignore-not-ignoring-my-files).

![image](https://user-images.githubusercontent.com/66967274/143009473-d49a95b7-81e8-495c-b326-22016465db2a.png)

Reply:
My Godness !!!
thanks, Vanessa
the previous reviewer did not mention how I should proceed to remove it from the repository but now am afraid to submit the last review... 

### can I send you a shot to see if I did fine before I request for the review, please?
![Screenshot from 2021-11-23 13-11-51](https://user-images.githubusercontent.com/77843050/143014590-155b8033-7cbe-49c9-bad7-24929dac5719.png)

Reply:
Great job! You should proceed and ask for a re-review!